### ISSUE TITLE
Mpdf fontDir should merge existing fontDir

### ISSUE THREAD

**Issue by lutek** [yii2tech/html2pdf#17]
docs: https://mpdf.github.io/fonts-languages/fonts-in-mpdf-7-x.html

from yii2tech\html2pdf\converters\Mpdf, this line should merged with defaults config from Mpdf (see docs) or should be removed. With one font dir i got errors about include defaults fonts
```php
if (isset($options['fontDir'])) {
                $config['fontDir'] = Yii::getAlias($options['fontDir']);
                unset($options['fontDir']);
            }
```

my call:
```php        

        $html = $this->render('pdf', [
            'model' => $model,
        ]);

        $defaultConfig = (new \Mpdf\Config\ConfigVariables())->getDefaults();
        $fontDirs = $defaultConfig['fontDir'];

        $defaultFontConfig = (new \Mpdf\Config\FontVariables())->getDefaults();
        $fontData = $defaultFontConfig['fontdata'];

        $path = Yii::getAlias('@app/data/pdf/');
        $targetFile = $path . $model->qr_key . '.pdf';

        $fontDir = Yii::getAlias('@app/data/fonts/Roboto');

        $options = [
            'fontDir' => array_merge($fontDirs, [
                $fontDir,
            ]),
            'fontdata' => $fontData + [
                    'Roboto' => [
                        'R' => 'Roboto-Regular.ttf',
                        'I' => 'Roboto-Italic.ttf',
                        'B' => 'Roboto-Bold.ttf',
                    ]
                ],
            'default_font' => 'Roboto'
        ];

            Yii::$app->html2pdf
                ->convertFile($path . 'pdf-template-raw.html', $options)
                ->saveAs($targetFile);
 
```

Error: 
```php
yii\base\ErrorException: strncmp() expects parameter 1 to be string, array given in /var/www/site/vendor/yiisoft/yii2/BaseYii.php:133
Stack trace:
#0 /var/www/site/vendor/yii2tech/html2pdf/src/converters/Mpdf.php(47): yii\BaseYii::getAlias()
#1 /var/www/site/vendor/yii2tech/html2pdf/src/BaseConverter.php(33): yii2tech\html2pdf\converters\Mpdf->convertInternal()
#2 /var/www/site/vendor/yii2tech/html2pdf/src/BaseConverter.php(41): yii2tech\html2pdf\converters\Mpdf->convert()
#3 /var/www/site/vendor/yii2tech/html2pdf/src/Manager.php(241): yii2tech\html2pdf\converters\Mpdf->convertFile()
#4 /var/www/site/controllers/DocumentController.php(521): yii2tech\html2pdf\Manager->convertFile()
#5 /var/www/site/vendor/yiisoft/yii2/base/InlineAction.php(57): app\controllers\DocumentController->actionPdf()
#6 /var/www/site/vendor/yiisoft/yii2/base/InlineAction.php(57): ::call_user_func_array:{/var/www/site/vendor/yiisoft/yii2/base/InlineAction.php:57}()
#7 /var/www/site/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams()
#8 /var/www/site/vendor/yiisoft/yii2/base/Module.php(528): app\controllers\DocumentController->runAction()
#9 /var/www/site/vendor/yiisoft/yii2/web/Application.php(103): yii\web\Application->runAction()
#10 /var/www/site/vendor/yiisoft/yii2/base/Application.php(386): yii\web\Application->handleRequest()
#11 /var/www/site/web/index.php(48): yii\web\Application->run()
#12 {main}
```

my versions:
        "mpdf/mpdf": "^7.1.0",
        "yii2tech/html2pdf": "~1.0.0",
        "yiisoft/yii2": "~2.0.16",
PHP 7.1

**Comment by lutek** [comment]
or add it by:
https://mpdf.github.io/reference/mpdf-functions/addfontdirectory.html only mPDF >= 7.0

**Comment by lutek** [comment]
i cant now create pull request but solution is:

```php

        if (class_exists('Mpdf\Mpdf')) {
            $config = [
                'mode' => $charset,
                'format' => $pageSize,
                'tempDir' => Yii::getAlias(ArrayHelper::remove($options, 'tempDir', '@runtime')),
            ];

            $pdf = new \Mpdf\Mpdf($config);

            if (isset($options['fontDir'])) {
                $pdf->AddFontDirectory(Yii::getAlias($options['fontDir']));
                unset($options['fontDir']);
            }

        } else {
            $pdf = new \mPDF($charset, $pageSize);
        }

```

**Comment by klimov-paul** [comment]
Resolved by commit c68115bfc4016cd4bc287037bfd17d9526100dce
